src/containers/share-sound-to-tiktok-modal/store/types.ts

```diff
@@ -0,0 +1,33 @@
+import { ID } from 'models/common/Identifiers'
+
+export enum Status {
+  SHARE_STARTED,
+  SHARE_SUCCESS,
+  SHARE_ERROR
+}
```

Comment:
Love how you modeled this as distinct states. This might be my own personal pref (and feel free to disagree), but rather than including status as Nullable and then having to deal with that in the typechecker + having the null state be less explicit than the other states, I like to just include an SHARE_UNINITIALIZED state or something and have the thing as non-optional

Reply:
Agreed, fixed!